.gitignore

```diff
@@ -3,4 +3,6 @@ felix.sh
 .classpath
 .project
 .settings/
-target/
\ No newline at end of file
+target/
+workspace/
+osgi-spring-felix/felix-cache
```

Comment:
Should we also add osgi-spring-felix/bundels to ignore list?

Btw, even though osgi-spring-felix/felix-cache is in ignore list, it still lists cache.lock in files changed list in the pull request.